llama-index-integrations/embeddings/llama-index-embeddings-cohere/llama_index/embeddings/cohere/base.py

```diff
@@ -115,9 +114,16 @@ class CohereEmbedding(BaseEmbedding):
         description="Embedding type. If not provided float embedding_type is used when needed."
     )
 
+    _client: cohere.Client = PrivateAttr()
+    _async_client: cohere.AsyncClient = PrivateAttr()
+    _base_url: Optional[str] = PrivateAttr()
+    _timeout: Optional[float] = PrivateAttr()
+    _httpx_client: Optional[httpx.Client] = PrivateAttr()
+    _httpx_async_client: Optional[httpx.AsyncClient] = PrivateAttr()
+
     def __init__(
         self,
-        cohere_api_key: Optional[str] = None,
```

Comment:
We should maybe keep (and mark as deprecated) the old kwarg to avoid a breaking change

Reply:
+1